libs/backend/src/cast_vote_record_report_import.test.ts

```diff
@@ -0,0 +1,163 @@
+import { assert } from '@votingworks/basics';
+import { electionMinimalExhaustiveSampleFixtures } from '@votingworks/fixtures';
+import { unsafeParse, CVR } from '@votingworks/types';
+import { CAST_VOTE_RECORD_REPORT_FILENAME } from '@votingworks/utils';
+import { rmSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import {
+  getCastVoteRecordReportImport,
+  validateCastVoteRecordReportDirectoryStructure,
+} from './cast_vote_record_report_import';
+import {
+  CVR_BALLOT_IMAGES_SUBDIRECTORY,
+  CVR_BALLOT_LAYOUTS_SUBDIRECTORY,
+} from './scan';
+
+const cdfCvrFile = electionMinimalExhaustiveSampleFixtures.standardCdfCvrFile;
+
+describe('getCastVoteRecordReportImport', () => {
+  test('imports a valid cast vote record report', async () => {
+    const castVoteRecordImportResult = await getCastVoteRecordReportImport(
+      join(cdfCvrFile.asDirectoryPath(), CAST_VOTE_RECORD_REPORT_FILENAME)
```

Comment:
I suspect all these temporary files will pile up fairly quickly. Maybe wrap this in something that cleans up after itself?

```ts
const cleanupPaths: string[] = [];

afterAll(() => {
  for (const path of cleanupPaths) {
    rmSync(path, { recursive: true });
  }
});

function makeCdfCvrFixture(): string {
  const path = cdfCvrFile.asDirectoryPath();
  cleanupPaths.push(path);
  return path;
}

Reply:
@eventualbuddha we could wrap it in this test file, but the problem will happen in other test files too. Perhaps if we use `tmp` in `res-to-ts` and `fixtures` then we can force cleanup? My understanding is that `tmp` has some ability to clean up after itself when a process ends that using the `fs.mktdirsync` does not.

Reply:
https://github.com/raszi/node-tmp#graceful-cleanup

Reply:
@eventualbuddha can you provide some feedback on my most recent commit as well, where I introduce some easy cleanup functionality? By putting copied fixtures into a subdirectory of `tmp`, we have an easy way to clean them all up at once. It's a bit clunky because it requires adding a `jest` teardown script and having a constant semi-shared between `res-to-ts` and `fixtures` (which could be one library anyway), but I like it because it offers a very simple way to cleanup fixtures across a test suite.